sdk/src/main/java/io/dapr/client/DaprClientBuilder.java

```diff
@@ -93,15 +91,16 @@ private DaprClient buildDaprClientHttp() {
         if (port <= 0) {
             throw new IllegalStateException("Invalid port.");
         }
-        if (daprHttClient == null) {
-            synchronized (okHttpClient) {
-                if (daprHttClient == null) {
-                    DaprHttp daprHtt = new DaprHttp(Constants.DEFAULT_HTTP_PROTOCOL_IDENTIFIED+host, port, okHttpClient);
-                    daprHttClient = new DaprClientHttpAdapter(daprHtt);
+        if (this.daprHttClient == null) {
+            synchronized (DaprClientBuilder.class) {
+                if (this.daprHttClient == null) {
+                    OkHttpClient okHttpClient = new OkHttpClient.Builder().build();
+                    DaprHttp daprHtt = new DaprHttp(Constants.DEFAULT_BASE_HTTP_URL, port, okHttpClient);
```

Comment:
Here was the only play where the variable host was actually used, with this approach we can remove that variable all togheter along with the validation for it.